tests/lifecycle/driver_test.go

```diff
@@ -23,8 +23,8 @@ func TestServices(t *testing.T) {
 	azureConfig, err := azure.GetConfigFromEnvironment()
 	assert.Nil(t, err)
 
-	catalogConfig, err := service.GetCatalogConfigFromEnvironment()
-	assert.Nil(t, err)
+	catalogConfig := service.NewCatalogConfigWithDefaults()
+	catalogConfig.MinStability = service.StabilityExperimental
```

Comment:
This is a programmatic override to the new default stability level of `PREVIEW`.